source/Calamari.Common/Features/Behaviours/StructuredConfigurationVariablesBehaviour.cs

```diff
@@ -18,7 +18,7 @@ public StructuredConfigurationVariablesBehaviour(IStructuredConfigVariablesServi
 
         public bool IsEnabled(RunningDeployment context)
         {
-            return context.Variables.GetFlag(KnownVariables.Package.JsonConfigurationVariablesEnabled);
+            return context.Variables.GetFlag(ActionVariables.StructuredConfigurationVariablesEnabled);
```

Comment:
I think you can delete `KnownVariables.Package.JsonConfigurationVariablesEnabled`